spec/chartmogul/task_spec.rb

```diff
@@ -0,0 +1,122 @@
+# frozen_string_literal: true
+
+require 'spec_helper'
+
+describe ChartMogul::Task do
+  let(:attrs) do
+    {
+      task_uuid: task_uuid,
+      customer_uuid: customer_uuid,
+      task_details: 'This is some task details text.',
+      assignee: 'keith+test1@chartmogul.com',
+      due_date: '2025-04-30T00:00:00.000Z',
+      completed_at: nil
+    }
+  end
+
+  let(:task_uuid) { '6c969fee-2179-11f0-8d8c-ef2a5afba642' }
+  let(:customer_uuid) { 'cus_9c8cc2bd-762e-4d93-ae34-1cfb53a53f64' }
+  let(:updated_attributes) { { task_details: 'This is updated task details text.' } }
+  let(:cursor) { 'MjAyNS0wNC0yNVQwMjowMTowOC4zNTM0NDEwMDBaJjI3NWJjMGJjLTIxNzktMTFmMC04OTdiLWYzZTcwZDJhYTU4ZA==' }
+
+  describe '#initialize' do
+    subject { described_class.new(attrs) }
+
+    it 'sets the read-only properties correctly' do
```

Comment:
[nitpick] you can just delete all the "correctly" since it adds 0 information value 😹